src/ui/filter/security.go

```diff
@@ -54,93 +78,175 @@ func SecurityFilter(ctx *beegoctx.Context) {
 	}
 
 	if !strings.HasPrefix(req.URL.RequestURI(), "/api/") &&
-		!strings.HasPrefix(req.URL.RequestURI(), "/service/") {
+		!strings.HasPrefix(req.URL.RequestURI(), "/service/token") {
```

Comment:
By default let the request pass through the modifiers.
Let's use a blacklist instead of a white list.